src/main/java/com/checkmarx/sdk/config/ScaProperties.java

```diff
@@ -28,4 +28,5 @@
     private String tenant;
     private String username;
     private String password;
+    private boolean enabledZipScan;
```

Comment:
Where is this property used? I couldn't find any usages.

Reply:
It's not used within the SDK but in the Cx-Flow.